language-extensions/java/src/linux/JavaExtensionUtils_linux.cpp

```diff
@@ -163,6 +163,25 @@ JavaExtensionUtils::fn_createJvm JavaExtensionUtils::LoadJvm(const string& jvmPa
 	return reinterpret_cast<fn_createJvm>(dlsym(g_jvmLib, "JNI_CreateJavaVM"));
 }
 
+//----------------------------------------------------------------------------
+// Name: JavaExtensionUtils::CleanupJvm
+//
+// Description:
+//  Cleans up JVM resources. Doesn't call Invocation API function
+//  DestroyJavaVM to promote faster SPEES query execution in Linux versions
+//  of SQL Server where calling that function contributed to a minimum
+//  5 second SPEES query execution duration.
```

Comment:
Maybe it's better if we have the comment about what was the the issue (another deamon does not let destryoing the JVM? or whatever you've found so far) and why it's safe to skip it (e.g. since once the processes exit, all the resources are cleaned up by OS). We can have a work item and also just put the work item id here (not the link to ADO). This can help future developers to find out why we skipped it and if they can fix it somehow.

5 seconds minimum time is a config related value related to how long we wait before timing out and we don't need to mention here I think.

Reply:
updated to 
> //  Cleans up JVM resources. Doesn't call Invocation API function
//  DestroyJavaVM since ExtHost process exit defers to OS for resource
//  cleanup and SPEES query doesn't hang.

Reply:
tracking further investigation in https://msdata.visualstudio.com/Database%20Systems/_workitems/edit/3090980